manifests/profile/mistral.pp

```diff
@@ -359,6 +367,20 @@
           content => template('st2/etc/init/mistral.conf.erb'),
           notify  => Service['mistral'],
         }
+        if $api_service {
```

Comment:
This looks right.

Tangental, iirc... do you need to change flags to the main service to prevent the `api_service` from running?